openmdao/drivers/latinhypercube_driver.py

```diff
@@ -47,7 +51,10 @@ def __init__(self, num_samples=1, seed=None, num_par_doe=1, load_balance=False):
     def _build_runlist(self):
         """Build a runlist based on the Latin Hypercube method."""
         design_vars = self.get_desvar_metadata()
-        self.num_design_vars = len(design_vars)
+
+        # Add up sizes
+        self.num_design_vars = sum(meta['size'] for _, meta in iteritems(design_vars))
```

Comment:
better to use itervalues(design_vars) here since the values are all you're using